src/AmplifyGeofenceControl/ui.ts

```diff
@@ -271,6 +271,12 @@ export function AmplifyGeofenceControlUI(
       "amplify-ctrl-list",
       geofenceListContainer
     );
+    _geofenceList.addEventListener("scroll", () => {
+      const { scrollHeight, scrollTop, clientHeight } = _geofenceList;
+      if (scrollTop + clientHeight >= scrollHeight - 20) {
```

Comment:
curious of what happens in an `else` case here. Main comment: Can you add a gif to show the loading of more data when a scroll occurs.

Reply:
Here if we're just not at the bottom of the page list then we just do nothing, or that's the intention anyways :)

Reply:
Added gif below